week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -19,22 +19,27 @@ You have to implement the missing features according to the specification.
 
 // Here is your starting point:
 let bottle = {
-  volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
-  },
-  pour: function () {
-    // calling this function should increase your bottle volume by 10 unit;
-  },
-  drink: function() {
-    // calling this function should decrease your bottle volume by 10 unit;
-  },
-  isFull: function () {
-    // this function should return true if your bottle is empty;
-  },
-  isEmpty: function() {
-    // this function should return true if your bottle is full;
-  }
+    volume: 0,
+    fillUp: function() {
+        // calling this function should pour your bottle full (volume = 100);
+        this.volume = 100;
+    },
+    pour: function() {
+        // calling this function should increase your bottle volume by 10 unit;
+        this.volume < 100 ? this.volume += 10 : this.volume
+    },
+    drink: function() {
+        // calling this function should decrease your bottle volume by 10 unit;
+        this.volume > 0 ? this.volume -= 10 : this.volume
+    },
+    isFull: function() {
+        // this function should return true if your bottle is empty;
+        return this.volume === 100 ? true : false

```

Comment:
something similar to alive issue
try to think what you want to return after the condition
if you are trying to return true and false, then no need for if statement, as the condition itself can be the boolean
```return this.volume ===100```

Reply:
done